src/libraries/System.Drawing.Common/src/System/Drawing/Icon.Windows.cs

```diff
@@ -380,6 +381,7 @@ private void DrawIcon(IntPtr dc, Rectangle imageRect, Rectangle targetRect, bool
             finally
             {
                 RestoreClipRgn(dc, hSaveRgn);
+                Interop.Gdi32.DeleteObject(hSaveRgn);
```

Comment:
This is going to cause merge conflicts with my PR: https://github.com/dotnet/runtime/pull/47836 which includes this change.

Reply:
Whoops. Forgot I still had that code in there.